src/app/chefs/ChefSchedule.jsx

```diff
@@ -18,14 +18,15 @@ class ChefSchedule extends React.Component {
     const { chefId } = this.props;
```

Comment:
if you're destructuring within the function signature I don't think you need to declare here

Reply:
I tried to remove it, and it breaks. So I'll leave it in.